### PR TITLE
Add master copy reindexing command

### PR DESCRIPTION
**Pull request by Uxio0**

### CODE REVIEW

File: safe_transaction_service/history/tasks.py [hunk 1]
```diff
@@ -127,15 +127,19 @@ def process_decoded_internal_txs_for_safe_task(self, safe_address: str) -> Optio
...
+                    # Find first corrupted safe status
+                    for safe_status in SafeStatus.objects.filter(address=safe_address).sorted_reverse_by_internal_tx():
+                        if safe_status.is_corrupted():
+                            message = f'A problem was found in SafeStatus with nonce={last_safe_status.nonce} ' \
+                                      f'on internal-tx-id={last_safe_status.internal_tx_id} ' \
+                                      f'tx-hash={last_safe_status.internal_tx.ethereum_tx_id} ' \
+                                      f'for safe-address={safe_address}, reindexing'
+                            logger.error(message)
+                            IndexServiceProvider().reprocess_addresses([safe_address])
+                            raise ValueError(message)
```
Comment (fmrsabino) [inline safe_transaction_service/history/tasks.py:142]: ❓ Is this related with the new command? Or what was the issue with this part?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ### What was wrong?
 
-Related to Issue #
+Closes issue #
 
 ### How was it fixed?
+
+@gnosis/safe-services
```
Comment (fmrsabino) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: nitpick: did you want to change this in this PR?

Comment (Uxio0) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: Yes, I didn't want to create a new PR for this 😅

File: safe_transaction_service/history/tasks.py (hunk 1 above)
Comment (Uxio0) [reply safe_transaction_service/history/tasks.py:142]: It's related. Previously when there was an indexing error it just reported when it was detected. Now it will report the exact tx when it was detected, so we can use the command to fix it. Maybe at some point it could be automatic

**Review by fmrsabino — APPROVED** [review]
